README.md

```diff
@@ -55,7 +55,8 @@ MongoConfig mongoConfig = MongoConfig.builder()
 		.connectionUri("mongodb://localhost:27017,localhost:27018,localhost:27019/?replicaSet=dbrs&retryWrites=true&w=majority")
 		.databaseName("test-db")
 		.collectionName("example")
-        .keyName("_id")
+		.match(Filters.lt("fullDocument.testValue", 2))
+		.keyName("_id")
```

Comment:
Please add a few more examples with (+ test for them):
 - more than one operation in .match
 - simple aggregation in .match
 
 Please include what operation from aggregation pipeline we support in .match method

Reply:
Created 1 more tests with multiple conditions. 
However, this parameter becomes part of `Aggregates.match()` so this is only for filtering. If we want to further [modify change stream output](https://www.mongodb.com/docs/manual/changeStreams/#modify-change-stream-output) we would have to accept more parameters or whole pipeline (`List<Bson>`) and somehow include our partition filtering in it.

Reply:
Is it possible to pass a transformation pipeline in there? Can we detect it somehow? I would add a couple of tests to check what happens then and make sure we only allow filtering

Reply:
It's possible to pass transformations because that's Bson too. There are Bson implementations like AndFilter, OperatorFilter, NotFilter but they are all private. I can't see any easy way to allow filters only, except for whitelisting with complex validation.
For example `addFields` results in:
> Command failed with error 2 (BadValue): 'unknown top level operator: $addFields' on server localhost:32793.

But maybe we could let user define whole pipeline (`List<Bson>`) and then find `match` stage and include  partitioning feature in it. What do you think?

Reply:
If it fails with transformations I think that's fine, I guess you just can't use any when defining change streams? If it works the same way a raw change stream would behave that's okay by my standards